.changelog/23973.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/aws_vpc_ipam: support aws_vpc_ipam cascade for easier deletes
```

Comment:
Can you please change this to "add `cascade` argument"

Reply:
sure